### PR TITLE
Add Index & CSS

### PR DESCRIPTION
**Pull request by hasenat-eqbali**
I have added index file with css in the project.

### CODE REVIEW

**Comment by hasenat-eqbali** [comment]
Required Changes ♻️

- It will be great to add Linter so you can check the errors easily and fix them in a minute. 👍
- I think it would be better to add the README file, so your user can understand your project. 😍
-
Thank You!❤️

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @hasenat-eqbali  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- Comments are polite ✅ 
- The Pull Request uses "Review changes"

## Suggested changes

- [ ] You did an amazing job reviewing the code :+1:, but as a requirement, you should add at least one comment using [GitHub inline comments feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)

- [ ] I think it would be better if your comments always include an explanation to help the student understand what you mean :+1:. You can check more about it [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate)

- [ ] Finally, as a requirement you should phrase the comments politely, as suggestions instead of orders.
  - This is a polite suggestion: *"I think it would be better to add the README file..."*
  - This reads as an order: *"Add Read me file with a good description."*

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @hasenat-eqbali   👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far 👍 
 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 
✔ Professional PR

## Required Changes ♻️

- [ ] **As suggested by the previous reviewer**, think it would be better to use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments. 👍 

- [ ]  As suggested by the previous reviewer, It would be better also to always include an explanation, a "why", for each requested change to make it clear to the code writer 😍😎

## Optional request 
- [ ] **[Optional]** I think It would be better to use a couple of emojis or gifs in the review to make the code review more human-friendly 👽😍

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @hasenat-eqbali 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


Required Changes ♻️

- [ ] Just as the previous reviews mentioned, you're meant to use inline comments to give a review, which should at least be two. the one you did happens to be a summary. Please check the [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) Section to see how this is done.  I've also included a screenshot from the guide to show you how. check them below 👍 

image


Check the comments under the review.

 I think you forgot to complete your two inline comments, I would advise you to always complete your comments so that the student can well understand your comment.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -1,12 +1,107 @@
...
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
 </head>
+
 <body>
-
```
Comment (hasenat-eqbali) [inline index.html:10]: Please use html5 structure to start your project. html5 is the latest version of HTML, and it makes your code more readable for the browser. it supports other languages as well.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,107 @@
...
+                <p>What is your favorite feature of freeCodeCamp?</p>
+                <select id="most-like" name="mostLike" class="form-control" required>
+                    <option disabled selected value>Select an option</option>
+                    <option value="challenges">Challenges</option>
+                    <option value="projects">Projects</option>
+                    <option value="community">Community</option>
+                    <option value="openSource">Open Source</option>
+                </select>
+            </div>
+
```
Comment (hasenat-eqbali) [inline index.html:63]: Kindly, remove extra blank lines and format your code like a pro. it makes your code more readable and easy to understand for review and debugging.

**Review by hasenat-eqbali — COMMENTED** [review]
Required changes 🧩

Please review the comments below. 

Thanks and happy coding.

**Review by amejid — APPROVED** [review]
Hi @hasenat-eqbali, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of Github features :heavy_check_mark: 
- Comments are polite :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**